run_ocsci.py

```diff
@@ -1,123 +1,15 @@
-    """
-    Function to init the default config for OCS CI
-
-    Args:
-        arguments (list): Arguments for pytest execution
-        default_config (str): Default config data
-
-    """
-    custom_config = get_param('--ocsci-conf', arguments)
-    cluster_config = get_param('--cluster-conf', arguments)
-    config_data = ocsci_config.CONFIG
-    with open(default_config) as file_stream:
-        default_config_data = yaml.safe_load(file_stream)
-    update_dict_recursively(config_data, default_config_data)
-    if custom_config:
-        with open(custom_config) as file_stream:
-            custom_config_data = yaml.safe_load(file_stream)
-        update_dict_recursively(config_data, custom_config_data)
-    if cluster_config:
-        with open(cluster_config) as file_stream:
-            cluster_config_data = yaml.safe_load(file_stream)
-        update_dict_recursively(config_data, cluster_config_data)
-    rendered_config = render_yaml_with_j2_context(config_data)
-    for key, value in rendered_config.items():
-        if key == "DEFAULTS":
-            for default_key, default_value in value.items():
-                setattr(defaults, default_key, default_value)
-        elif key.isupper():
-            setattr(ocsci_config, key, value)
-
-
-def update_dict_recursively(d, u):
-    """
-    Update dict recursively to not delete nested dict under second and more
-    nested level. This function is changing the origin dictionary cause of
-    operations are done on top of it and dict is a mutable object.
-
-    Args:
-        d (dict): Dict to update
-        u (dict): Other dict used for update d dict
-
-    Returns:
-        dict: returning updated dictionary (changes are also done on dict `d`)
-    """
-    for k, v in u.items():
-        if isinstance(d, collections.Mapping):
-            if isinstance(v, collections.Mapping):
-                r = update_dict_recursively(d.get(k, {}), v)
-                d[k] = r
-            else:
-                d[k] = u[k]
-        else:
-            d = {k: u[k]}
-    return d
-
-
-def render_yaml_with_j2_context(yaml_data):
-    """
-    Render yaml template with own context.
-
-    Args:
-        yaml_data (dict): Yaml data
-
-    Returns:
-        dict: rendered data
-    """
-    template = Template(yaml.dump(yaml_data))
-    out = template.render(**yaml_data)
-    return yaml.safe_load(out)
+import ocsci.main
 
 
 def main():
-    arguments = sys.argv[1:]
-    init_ocsci_conf(arguments)
-    arguments.extend([
-        '-p', 'ocsci.pytest_customization.ocscilib',
-        '-p', 'ocsci.pytest_customization.marks',
-        '-p', 'ocsci.pytest_customization.ocsci_logging',
-    ])
+    HERE = os.path.abspath(os.path.dirname(__file__))
     from utility.utils import add_path_to_env_path
     add_path_to_env_path(os.path.join(HERE, 'bin'))
-    exit(pytest.main(arguments))
+    arguments = sys.argv[1:]
+    exit(ocsci.main.main(arguments))
```

Comment:
I would double check issues similar to https://github.com/red-hat-storage/ocs-ci/issues/233 (test that one can install both with clean state and then with cli tools already placed in bin directory).

Reply:
I did test that